cmd/in/main.go

```diff
@@ -2,25 +2,31 @@ package main
 
 import (
 	"encoding/json"
+	"io/ioutil"
 	"os"
-
-	"github.com/zabawaba99/stash-commit-status-resource/resource"
 )
 
 func main() {
-	var req resource.Request
-	if err := json.NewDecoder(os.Stdin).Decode(&req); err != nil {
-		resource.Error("Could not unmarshal request %s", err)
+	var out struct {
+		Version interface{} `json:"version"`
 	}
 
-	if req.Version.Ref == "" {
-		req.Version.Ref = "1"
+	inBytes, err := ioutil.ReadAll(os.Stdin)
+	if err != nil {
+		panic(err)
```

Comment:
Instead of panicking, what do you think about using the `resource.Error` function since it already prints and exits. It would keep it consistent with the other commands in this resource.